README.md

```diff
@@ -67,6 +67,13 @@ For container runtime Containerd:
 kubectl apply -f https://raw.githubusercontent.com/logzio/logzio-k8s/master/logzio-daemonset-containerd.yaml -f https://raw.githubusercontent.com/logzio/logzio-k8s/master/configmap.yaml
 ```
 
+For monitoring fluentd:
+add the suffix `-monitoring` to the desired configmap and daemonset templates.
```

Comment:
I think this instruction may be confusing. Maybe it should explicitly say something like "To monitor fluentd on cluster with containerd cri - use the following ............. for monitor fluentd on cluster with docker cri - use the following......."